e2e/installs.test.js

```diff
@@ -46,9 +72,9 @@ describe('Installation', () => {
       execSync(
         `yarn start ${appPath} \
           --name ${appName} \
-          --template "InstantSearch.js" \
-          --no-installation`,
-        { stdio: 'ignore' }
+          --config ${configFilePath} \
+          --no-installation`
+        // {stdio: 'ignore'}
```

Comment:
Should we remove the `ignore` option in all the E2E commands?

Reply:
ah this should actually be added again